assets/javascripts/discourse/components/ai-artifact.gjs

```diff
@@ -98,11 +113,38 @@ export default class AiArtifactComponent extends Component {
     }
   }
 
+  get heightStyle() {
+    if (this.args.artifactHeight) {
+      let height = parseInt(this.args.artifactHeight, 10);
```

Comment:
I wonder if there's any payload that can tryp parseInt here, nothing that I tried worked